src/Proxy.hs

```diff
@@ -1698,65 +1756,70 @@ instance FromJSON DropSchedule where
       return Normal
     else fail "Unsupported drop type."
 
--- | Handle a GTU drop request.
---     When 'gtuDropData' is provided: handle drop.
---     Otherwise: return 404.
+-- |Return a handler which GTU drop.
+-- Attempt to make a GTU drop to the account with the specified address if @gtuDropData@ was provided
+-- in the environment settings. Responds with a 404-status code if @gtuDropData@ was absent.
 putGTUDropR :: Text -> Handler TypedContent
 putGTUDropR addrText = do
-    i <- internationalize
     Proxy{..} <- getYesod
     case gtuDropData of
+      -- VH/FIXME: Should this be 404?
       Nothing -> respond404Error $ EMErrorResponse NotFound
```

Comment:
Should this be 404?

Reply:
Yeah, the endpoint does not exist is the intention. So 404 is appropriate.